### PR TITLE
feat: GovTech certificate of achievement

### PR DESCRIPTION
**Pull request by gjj**
- CSS cleanup
- updated README status badge
- updated sample certificate structure

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,6 +10,7 @@
     "@govtechsg/decentralized-renderer-react-components": "^2.3.0",
     "@hot-loader/react-dom": "^16.11.0",
     "debug": "4.1.1",
+    "moment": "^2.27.0",
```
Comment (Nebulis) [inline package.json:13]: let's use date-fns instead (I let you google the why :))

File: src/templates/certificateOfAchievement/certificateOfAchievement.tsx [hunk 2]
```diff
@@ -3,26 +3,22 @@ import { TemplateProps } from "@govtechsg/decentralized-renderer-react-component
 import { css } from "@emotion/core";
 import { GovTechCertificateTemplate } from "../sample";
 import certificateBase from "../../core/certificate-base.png";
+import moment from "moment";
```
Comment (Nebulis) [inline src/templates/certificateOfAchievement/certificateOfAchievement.tsx:6]: it's not used ?

File: package.json (hunk 1 above)
Comment (gjj) [reply package.json]: https://javascript-conference.com/blog/why-you-should-use-date-fns-for-manipulating-dates-with-javascript/ ?

TIL date-fns exists.. always used moment.js hahah thanks!

File: src/templates/certificateOfAchievement/certificateOfAchievement.tsx (hunk 2 above)
Comment (gjj) [reply src/templates/certificateOfAchievement/certificateOfAchievement.tsx]: removed!

File: package.json (hunk 1 above)
Comment (Nebulis) [reply package.json]: I don't see it uses, normal?

Comment (gjj) [reply package.json]: ah yeah I was going to do the date formatting feature on a separate branch, then I forgot to uninstall from here...

anyway it's no longer in this repo!

**Pull request merged**